bucket/deltachat.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.3.4",
+    "description": "Email-based secure instant messaging",
+    "homepage": "https://delta.chat",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.delta.chat/desktop/v1.3.4/DeltaChat%201.3.4.exe#/dl.7z",
+            "hash": "9f6f1db789a8ca2a59e2c6c7423d6d128892b4ec038537a4a3cc22f1110d8b20",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse"
+                ]
+            }
+        }
+    },
+    "bin": "DeltaChat.exe",
```

Comment:
What are available parameters??

Reply:
None. Just for consistence with other manifest.

Reply:
Then remove it.